README.md

```diff
@@ -104,7 +104,12 @@ For a Rails application this would go in `config/initializers/webauthn.rb`.
 WebAuthn.configure do |config|
   # This value needs to match `window.location.origin` evaluated by
   # the User Agent during registration and authentication ceremonies.
-  config.origin = "https://auth.example.com"
+  # Multiple origins can be used in more complex scenarios (e.g. different subdomains, native client sending android:apk-key-hash:... etc.)
+
+  config.allowed_origins = [
+    "https://auth.example.com",
+    "android:apk-key-hash:blablablablablalblalla"
+  ]
```

Comment:
I'd leave this example configuration using one single element (the first one) because I'm assuming that is the majority of our gem users use cases.